### ISSUE TITLE
Incompatible with BullMQ v2?

### ISSUE THREAD

**Issue by smonn** [felixmosh/bull-board#463]
We recently upgraded to `bullmq` v2.1.3 and `@bull-board/fastify` v4.3.2. For most queues this seems fine, but some error out with this in the console:

```
Failed to poll Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7Bdata%2C%20opts%2C%20id%2C%20progress%2C%20returnvalue%2C%20stacktrace%2C%20attemptsMade%2C%20delay%2C%20timestamp%2C%20finishedOn%2C%20processedOn%2C%20failedReason%7D for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
```

Is Bullboard compatible with BullMQ v2? Let me know if any other details would be helpful.

**Comment by felixmosh** [comment]
I've not tested it yet, so probably there is issue.
But if it is related to a specific queue, this probably an issue with some data, If you will able to identify the object that is passed as React component, it will help a lot.

Thanx

**Comment by smonn** [comment]
It looks like there were some weird jobs in some of our queues with job ID `0`. I saw there was this change in v2 https://github.com/taskforcesh/bullmq/commit/829e6e0252e78bf2cbc55ab1d3bd153faa0cee4c

Not sure if it's due solely to that, but after manually using the bull-board API to clear those jobs everything seems back to normal. For future reference, we executed this for all queues with this issue:

```http
PUT /bullboard/api/queues/:queue/0/clean
```

Looks like we're back to normal now. Wouldn't have been this easy to recover without bull-board, so thank you for that! Closing this as I'm pretty confident the root cause was the upgrade to BullMQ v2 and not related to bull-board.
